dags/atd_executive_dashboard_row_weekly_summary.py

```diff
@@ -166,4 +166,17 @@
         retry_delay=duration(seconds=60),
     )
 
-    t1 >> t2 >> t3 >> t4 >> t5
+    t6 = DockerOperator(
+        task_id="amanda_review_time",
+        image=docker_image,
+        auto_remove=True,
+        command=f"python AMANDA/amanda_to_s3.py --query review_time",
```

Comment:
Just out of curiosity, will this create a new csv called review_time.csv in the `atd-exec-dashboard`. I peeked the bucket and wanted to check my understanding of the output of the script. 🙏

Reply:
yes that should happen 🤞!